synapse/rest/admin/media.py

```diff
@@ -328,3 +488,4 @@ def register_servlets_for_media_repo(hs: "HomeServer", http_server: HttpServer)
     ListMediaInRoom(hs).register(http_server)
     DeleteMediaByID(hs).register(http_server)
     DeleteMediaByDateSize(hs).register(http_server)
+    UserMediaRestServlet(hs).register(http_server)
```

Comment:
Weirdly I think this means this will be available when a `client` listener is configured since `register_servlets_for_media_repo` is called via `register_servlets_for_client_rest_resource`.

I don't think that's a huge deal, although we should probably clean all that up at some point...